.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  workflow_dispatch:
+
+jobs:
+  build_and_package:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-2019, macos-latest]
```

Comment:
needed because xv cant be built on the last version of msvc, fix is here though: https://gitlab.kitware.com/paraview/common-superbuild/-/merge_requests/533